int64 imgID
---
bool success